src/Bicep.Core/Bicep.Core.csproj

```diff
@@ -33,7 +33,7 @@
     <PackageReference Include="Azure.Deployments.Templates" Version="1.0.815" />
     <PackageReference Include="Azure.Deployments.Expression" Version="1.0.815" />
     <PackageReference Include="Azure.Bicep.Types" Version="0.3.99" />
```

Comment:
Don't we need to update Azure.Bicep.Types and Azure.Bicep.Types.K8s?

Reply:
Not sure. Happy to do it

Reply:
@shenglol, @jeskew the instructions here is only Azure.Bicep.Types.Az and needs to be updated to include Azure.Bicep.Types and Azure.Bicep.Types.K8s as well.

We do need to update all three of them every release correct?

Reply:
Chatted online with @davidcho23. We don't need the other two.